test/framework/sandbox/sources/toy2/toy2-0.0/toy2.c

```diff
@@ -0,0 +1,6 @@
+#include <toy.h>
+
+int main(int argc, char* argv[]){
+    toy_function();
```

Comment:
Add a comment here that this is provided by the `libtoy` library?

Maybe we should also use a better name that `toy2`... How about `toy-cli` or `toy-app`?

Reply:
Done